compiler/one-cmds/onnx_legalizer.py

```diff
@@ -1040,8 +1052,14 @@ def legalize(model, options):
 
 if __name__ == '__main__':
     if len(sys.argv) < 3:
-        print('usage: ./legalize_onnx.py <path to input model> <path to output model>')
+        print('usage: ./legalize_onnx.py <path to input model> <path to output model>\n'
+              '\n'
+              '    In stand-alone utility mode this tool provides basic funtionality\n'
+              '    If you want to have more control over applied transformations, use this legalizer as a library')
         exit(1)
+    options = LegalizeOptions()
+    options.unroll_lstm = True
+    options.unroll_rnn = True
     model = onnx.load(sys.argv[1])
-    legalize(model)
+    legalize(model, options)
```

Comment:
Added options here, to make it possible run this functionality as an stand-alone program

Reply:
@binarman (optional) Isn't it inconvenient to use python script as both of library and stand-alone? I mean, it would have better maintainability to split into separate file - a file that has method as a library and another one for main. Seems to me that this script is too big to me.

Reply:
@mhs4670go 
Hmm, I am using script as a stand-alone tool simply for debug/experiment purposes, so maybe it is a good idea to remove or move it in separate file :thinking: 

About breaking this file in parts - it is probably a good idea, if we plan to extend it =)